aws/resources/route53_cidr_collection.go

```diff
@@ -0,0 +1,108 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/report"
+)
+
+func (r Route53CidrCollection) getAll(c context.Context, configObj config.Config) ([]*string, error) {
```

Comment:
I was wondering why is used value instead of pointer like in other cases like:

https://github.com/gruntwork-io/cloud-nuke/blob/master/aws/resources/ec2.go#L27